### ISSUE TITLE
Adding Tiffany Pender to DSVA Slack

### ISSUE THREAD

**Issue by anerbovig** [department-of-veterans-affairs/va.gov-team#42085]
Hi there! Please add tiffany.pender@adhocteam.us as a full member of DSVA Slack. Tiffany will be working as an accessibility specialist and supporting the VA Profile, Authenticated Experience, Debt Resolution, and CTO Health Tools teams. Thank you!

**Comment by anerbovig** [comment]
No worries, thanks Lacey!
